schema/workbook.json

```diff
@@ -1,2 +1,754 @@
-{    
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+    "type": "object",
+    "title": "Workbook template gallery schema",
+    "required": [
+        "version",
+        "items"
+    ],
+    "properties": {
+        "version": {
+            "$id": "#/properties/version",
+            "type": "string",
+            "title": "Workbook version",
+            "default": "Notebook/1.0",
+            "pattern": "^(.*)$"
+        },
+        "items": {
+            "$id": "#/properties/items",
+            "type": "array",
+            "title": "Items in the template",
+            "items": [
+                {
+                    "anyOf": [
+                        {
+                            "$ref": "#/definitions/markdown"
+                        },
+                        {
+                            "$ref": "#/definitions/metric"
+                        },
+                        {
+                            "$ref": "#/definitions/parameter"
+                        },
+                        {
+                            "$ref": "#/definitions/query"
+                        },
+                        {
+                            "$ref": "#/definitions/link"
+                        }
+                    ]
+                }
+            ]
+        },
+        "styleSettings": {
+            "$id": "#/properties/styleSettings",
+            "type": "object",
+            "title": "Workbook style settings"
+        },
+        "defaultResourceIds": {
+            "$id": "#/properties/defaultResourceIds",
+            "type": "array",
+            "description": "top level resources of the workbook, restored when the workbook is loaded"
+        },
+        "fallbackResourceIds": {
+            "$id": "#/properties/fallbackResourceIds",
+            "type": "array",
+            "description": "resourceIds that are opened with the workbook. Used if the workbook is loaded later from a place * with no explicit context passed in (like from a share link or a pin)"
+        },
+        "$schema": {
+            "$id": "#/properties/$schema",
+            "type": "string",
+            "title": "$schema",
+            "default": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+            "examples": [
```

Comment:
for schema, isn't just `default` field good enough`?  shouldn't need any example/pattern as isn't this technically the only legit value?